image_proc/src/edge_aware.cpp

```diff
@@ -39,8 +39,8 @@
 #define WAVG4(a,b,c,d,x,y)  ( ( ((int)(a) + (int)(b)) * (int)(x) + ((int)(c) + (int)(d)) * (int)(y) ) / ( 2 * ((int)(x) + (int(y))) ) )
 using namespace std;
```

Comment:
For the love of Pete, please fix this.

Reply:
i don't know how to change this, do you mean this code is too long?